docs/examples.md

```
@@ -394,5 +394,77 @@ True
 True
 ```
 
-!!! danger "Deprecated"
-    In the future
+## Generate iCalendar content and export to `.ics`
+
+[ICalExporter][holidays.ical.ICalExporter] facilitates the creation and export of iCalendar files
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "holidays.ical.icalexporter"

(MD052, reference-links-images)